test/testenv/verificationutils.go

```diff
@@ -684,7 +683,7 @@ func VerifyAppsInFolder(deployment *Deployment, testenvInstance *TestEnv, ns str
 		for _, app := range apps {
 			folderName := app + "/"
 			found := CheckStringInSlice(appList, folderName)
-			logf.Log.Info("Check App in Directory", "Pod Name", podName, "App-name", folderName, "Path", path, "status", found)
+			logf.Log.Info("App check", "On pod", podName, "check app", folderName, "is in path", path, "Status", found)
 			if found != checkAppDirectory {
```

Comment:
Do we need similar log updated in `VerifyAppsDownloadedOnContainer` ?

Reply:
I will handle that update as part of my change.